Standards/scs-XXXX-w1-Availability-Zones-Standard.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "SCS Availability Zone Standard: Implementation and Testing Notes"
+type: Supplement
+track: IaaS
+status: Draft
+supplements:
+  - scs-XXXX-vN-Availability-Zones-Standard.md
+---
+
+## Automated Tests
+
+The SCS will also allow small deployments and edge deployments, that both will not meet the requirement for bein divided into multiple Availability Zones.
+Thus Availability Zones are not always present and there will be no automated tests to search for AZs.
```

Comment:
What does it mean that Availability Zones are not always present? Does OpenStack distinguish between the cases (a) that there are no AZs and (b) that there is exactly one AZ?